validators/denali/denali-v2-upgrade.md

```diff
@@ -0,0 +1,92 @@
+## Denali V2 Upgrade Steps
+
+For SKALE Validator CLI initialization use SKALE Manager ABI:
+
+`https://raw.githubusercontent.com/skalenetwork/skale-network/master/releases/mainnet/skale-manager/1.8.1/skale-manager-1.8.1-mainnet-abi.json`
+
+### Update sgx
+
+Upgrade sgx wallet to `skalenetwork/sgxwallet_release:1.77.0-stable.0`.
```

Comment:
Add: Make sure to have sgxwallet_release:1.77.0-stable.0 in the docker-compose.yml